src/overlays/actors/ovl_En_Pamera/z_en_pamera.c

```diff
@@ -105,7 +105,25 @@ static CollisionCheckInfoInit2 sColChkInfoInit2 = {
     0, 0, 0, 0, MASS_IMMOVABLE,
 };
 
-static AnimationInfo sAnimationInfo[] = {
+typedef enum {
+    /*  -1 */ PAMELA_ANIM_NONE = -1,
+    /* 0x0 */ PAMELA_ANIM_0,
+    /* 0x1 */ PAMELA_ANIM_1,
+    /* 0x2 */ PAMELA_ANIM_2,
+    /* 0x3 */ PAMELA_ANIM_3,
+    /* 0x4 */ PAMELA_ANIM_4,
+    /* 0x5 */ PAMELA_ANIM_5,
+    /* 0x6 */ PAMELA_ANIM_6,
+    /* 0x7 */ PAMELA_ANIM_7,
+    /* 0x8 */ PAMELA_ANIM_8,
+    /* 0x9 */ PAMELA_ANIM_9,
+    /* 0xA */ PAMELA_ANIM_10,
+    /* 0xB */ PAMELA_ANIM_11,
+    /* 0xC */ PAMELA_ANIM_12,
+    /* 0xD */ PAMELA_ANIM_MAX
+} PamelaAnimation;
+
+static AnimationInfo sAnimationInfo[PAMELA_ANIM_MAX] = {
     { &object_pamera_Anim_0005BC, 1.0f, 0, 0.0f, ANIMMODE_LOOP, -4.0f },
     { &object_pamera_Anim_008AE0, 1.0f, 0, 0.0f, ANIMMODE_LOOP, -4.0f },
     { &object_pamera_Anim_008E38, 1.0f, 0, 0.0f, ANIMMODE_LOOP, -4.0f },
```

Comment:
Could you add comments with the enum values on this array please?
```suggestion
static AnimationInfo sAnimationInfo[PAMELA_ANIM_MAX] = {
    { &object_pamera_Anim_0005BC, 1.0f, 0, 0.0f, ANIMMODE_LOOP, -4.0f }, // PAMELA_ANIM_0
    { &object_pamera_Anim_008AE0, 1.0f, 0, 0.0f, ANIMMODE_LOOP, -4.0f }, // PAMELA_ANIM_1
    { &object_pamera_Anim_008E38, 1.0f, 0, 0.0f, ANIMMODE_LOOP, -4.0f }, // PAMELA_ANIM_2